RecoMuon/MuonIdentification/plugins/MuonIdProducer.cc

```diff
@@ -876,6 +888,14 @@ void MuonIdProducer::fillMuonId(edm::Event& iEvent, const edm::EventSetup& iSetu
      matchedChamber.edgeY = chamber.localDistanceY;
 
      matchedChamber.id = chamber.id;
+
+     if (fillShowerDigis_) {
+       theShowerDigiFiller_->fill(matchedChamber);
+     }
+     else {
+       matchedChamber.nDigisInRange = 0;
```

Comment:
The pattern above looks like you want to encapsulate setting of potentially more than one parameter; if that's the case,
perhaps ```theShowerDigiFiller_->fillDefault``` is more appropriate.
If there is no plan to add more variables, and only nDigisInRange is going to be computed, then instead of a ```fill(matchedChamber)`` there better be just nDigisInRange computation

Reply:
In the offline studies linked as slides at the beginning of the PR I had 4 ranges. I initially implemented this code thinking about 2 ranges, then I switched to one for simplicity. Let me briefly discuss with MUO and, depending on the outcome of the discussion I will change things accordingly.

Reply:
I discussed with @Fedespring and we agreed that, depending on the development of the studies being performed for MUO-17-001, the plan is very likely to extend this shower filler (potentially also replacing `MuonShowerInformationFiller`), hence I adopted the solution of adding a `theShowerDigiFiller_->fillDefault()` method.